### ISSUE TITLE
GTK3 branch builds GTK2 by default

### ISSUE THREAD

**Issue by diabl0w** [debrouxl/tilp_and_gfm#77]
I don't think the GTK3 branch builds GTK2 by default via cmake.
There is [some logic](https://github.com/debrouxl/tilp_and_gfm/blob/c241fcff0e44a7ee8ddec43a46db7bb408cac34f/tilp/trunk/CMakeLists.txt#L58) for doing so, but I don't believe that option gets properly set anywhere to GTK3.

**Comment by diabl0w** [comment]
calling cmake -GNinja -DBUILD_USING_GTK3=ON allows it to build fine with cmake, but this should be the default for this branch without requiring it to be specified

**Comment by adriweb** [comment]
Yes, once the GTK3 support is considered stable and ready it'll become the default (and the branch merged anyway)

**Comment by myclevorname** [comment]
Can we have a tracking issue for GTK+3?
